tutorial/create-test-virtual-device.md

```diff
@@ -0,0 +1,212 @@
+(tut-create-virtual-device)=
+# Create and test a virtual device
+
+> Before you start, complete: {ref}`tut-installing-appliance` but when initializing, remember to answer *Yes* to the question: *Do you want to setup access to the AMS API for your current user (ubuntu, uid=1000)?*
+
+In this tutorial, we will go through the first steps of using Anbox Cloud to create a virtual Android device. By *virtual device*, we mean a simulated device that runs a plain Android operating system without any special apps installed. Anbox Cloud treats such a virtual device as an *empty* application, meaning an application that is not running a specific APK.
+
+By the end of this tutorial, we will be familiar with an {term}`Application` and an {term}`Instance` in Anbox Cloud that are used to create and stream the virtual Android device.
+
+This tutorial has two paths - you can use the CLI or the dashboard, depending on your learning goals. The CLI is more powerful and gives you access to all features of Anbox Cloud, while the dashboard is a simpler user interface. For this tutorial, it does not matter which path you choose.
+
+`````{tabs}
+````{group-tab} CLI
+
+We will use the {term}`Anbox Management Client (AMC)` in this tutorial to work with applications and instances. AMC communicates with the {term}`Anbox Management Service (AMS)`, the management module of Anbox Cloud.
+
+## Create the device
+
+The first step is to create a `yaml` file that can be used as the application manifest. It is important that this file is created in your `home` directory.
+
+In the home directory, let's first create a directory for the application content:
+
+      mkdir my-app
+
+This directory is the location where the application manifest file and any relevant APK must reside.
```

Comment:
and any relevant APK must reside.
->
As an entry-level user, it might be assumed that the application can install multiple APKs, but the AMS only looks for the "app.apk" file within the specified directory. This could lead to confusion for the end user.
I'd suggest `any application data` or `any application artifacts`